tests/start_proxy/start_proxy_test.py

```diff
@@ -129,7 +129,7 @@ def test_gen_proxy_config(self):
               '--ssl_server_cert_path', '/etc/nginx/ssl',
               '--listener_port', '443',
               ]),
-            # ssl_backend_path specified
+            # ssl_client_cert_path specified
             (['-R=managed','--listener_port=8080',  '--disable_tracing',
               '--ssl_client_cert_path=/etc/endpoint/ssl'],
```

Comment:
Add all the error cases for flag conflicts.

Reply:
Done